dashboard/app/models/ai_tutor_interaction_feedback.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: ai_tutor_interaction_feedbacks
+#
+#  id                      :bigint           not null, primary key
+#  ai_tutor_interaction_id :bigint           not null
+#  user_id                 :integer          not null
+#  thumbs_up               :boolean
+#  thumbs_down             :boolean
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#
+# Indexes
+#
+#  index_ai_tutor_feedback_on_interaction_and_user  (ai_tutor_interaction_id,user_id) UNIQUE
+#
+class AiTutorInteractionFeedback < ApplicationRecord
+  belongs_to :ai_tutor_interaction
+  belongs_to :user
+
+  validates :thumbs_up, inclusion: {in: [true, false]}, allow_nil: true
```

Comment:
for my understanding: why do we need this given that `thumbs_up` and `thumbs_down` are nullable booleans?

Reply:
Since `thumbs_up` and `thumbs_down` are nullable booleans in the schema, you mean? I think the answer is -- we don't need these. They're mainly redundant (though they do prevent someone from having to look at the schema to know what these should be). Do you have a preference to remove? I'm good either way 🤷‍♀️

Reply:
yeah I think we don't need it? It makes sense to me to remove but I'm not a ruby expert.